packages/boot/test/fast-usdc/fast-usdc.test.ts

```diff
@@ -503,14 +509,6 @@ test.serial('LP withdraws', async t => {
   );
 });
 
-test.serial('restart contract', async t => {
```

Comment:
The change above is good. I think as a matter of course we should name this test in each suite so it's clear that it's tested.

So please leave this here (with incarnationNumber: 2 and a comment as to why)

A bit ouf of scope but I think we also here should verify that not only does the contract keep working (as above) but things that are expected to change on restart do. I.e. `feeConfig` in the private args should show up in vstorage. (see `writes fee config to vstorage` test).

Please add that to this PR or make a separate issue to do that.

Reply:
Made an issue https://github.com/Agoric/agoric-sdk/issues/10948

I'll do that as part of this PR then re-request review

Reply:
PTAL